sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -89,7 +89,7 @@ export class ServiceBusClient {
 
     const timeoutInMs = this._clientOptions.retryOptions.timeoutInMs;
     if (
-      timeoutInMs != undefined &&
+      timeoutInMs !== undefined &&
```

Comment:
Same question.

Reply:
Updated using the approach @chradek has for Event Hubs. See https://github.com/Azure/azure-sdk-for-js/pull/13935/commits/29e0618fd3e30f31f760eacf909aa9c2cdfce6b2